### PR TITLE
Newsletter/mailing list archives

### PR DESCRIPTION
**Pull request by joonas-somero**
[Archives listed on 'Contact' page](https://csc-guide-preview.rahtiapp.fi/origin/newsletter-archive/support/contact/#archives)

### CODE REVIEW

**Comment by joonas-somero** [comment]
The FIXME (should be the only thing failing the tests) will need to be removed before merging to master.

File: mkdocs.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ site_url: "https://docs.csc.fi/"
 site_description: Instructions and user guides for the CSC supercomputers, cloud services, storage and software, including FAQ and tutorials
 
 extra:
+  environment: !ENV [MKDOCS_ENV, "production"] # FIXME to this: environment: !ENV [MKDOCS_ENV, "preview"]
```
Comment (rkronberg) [inline mkdocs.yml:8]: Comment still relevant?

**Review by rkronberg — APPROVED** [review]
One small question about a comment. Otherwise great work! Btw, why was the FIXME required to force tests to fail?

File: mkdocs.yml (hunk 1 above)
Comment (joonas-somero) [reply mkdocs.yml:8]: The comment is what I'll replace the key with momentarily.

**Comment by joonas-somero** [comment]
> One small question about a comment. Otherwise great work! Btw, why was the FIXME required to force tests to fail?

I wanted to not have the tests pass until the workaround to show the archives in Preview (for the review) is removed, so I added a FIXME. Well, actually the tests aren't looking for FIXMEs in _mkdocs.yml_, so I added another one in _support/contact.md_. Anyway, I'll remove them now, and with that, the archives are only fetched when an environment variable `MKDOCS_ENV=production` is defined.

**Pull request merged**